clients/ops-l1/Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM ethereum/client-go:v1.10.16 AS base
+
+RUN apk add --no-cache jq yarn git linux-headers gcompat
+RUN /usr/local/bin/geth console --exec 'console.log(admin.nodeInfo.name)' --maxpeers=0 --nodiscover --dev 2>/dev/null | head -1 > /version.txt
+ENV GLIBC_KEY=https://alpine-pkgs.sgerrand.com/sgerrand.rsa.pub
+ENV GLIBC_KEY_FILE=/etc/apk/keys/sgerrand.rsa.pub
+ENV GLIBC_RELEASE=https://github.com/sgerrand/alpine-pkg-glibc/releases/download/2.35-r0/glibc-2.35-r0.apk
+RUN wget -q -O ${GLIBC_KEY_FILE} ${GLIBC_KEY} \
+    && wget -O glibc.apk ${GLIBC_RELEASE} \
+    && apk add glibc.apk --force
+
+# build forge
+from alpine as build-environment
+WORKDIR /opt
+RUN apk add clang lld curl build-base linux-headers git \
+    && curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs > rustup.sh \
+    && chmod +x ./rustup.sh \
+    && ./rustup.sh -y
+RUN git clone https://github.com/foundry-rs/foundry /opt/foundry
+WORKDIR /opt/foundry
+RUN ls /opt/foundry
+RUN source $HOME/.profile && cargo build --release \
```

Comment:
I don't think you have to build Foundry here. You could do:

```
FROM ghcr.io/foundry-rs/foundry:latest as foundry

COPY --from=foundry /opt/foundry/target/release/forge /usr/local/bin/forge
COPY --from=foundry /opt/foundry/target/release/cast /usr/local/bin/cast
```

Reply:
👍🏽

Reply:
Done.